sqlalchemy_imageattach/stores/fs.py

```diff
@@ -246,6 +255,9 @@ class StaticServerMiddleware(object):
     :type dir_path: :class:`str`
     :param block_size: the block size in bytes
     :type block_size: :class:`numbers.Integral`
+    :param cors: whether or not to allow the `Cross-Origin Resource Sharing`_
```

Comment:
[This line](https://github.com/dahlia/sqlalchemy-imageattach/pull/40/files/8a37976ca8c7057f921e23e6644c8d467aebe708#diff-c04c166fbb31a2272086ee3a58a8e5a2R175) is the reference.